### ISSUE TITLE
Consider supporting multiple AWS profiles via `--profile`

### ISSUE THREAD

**Issue by jen20** [dtan4/terraforming#86]
Working with multiple AWS accounts, I have the AWS CLI configured with different profiles and switch amongst them by changing the `AWS_DEFAULT_PROFILE` environment variable. It would be good if terraforming respected this setting (which encompasses the keys and the region) in addition to the environment variables.

**Comment by dtan4** [comment]
If you're using OS X, I recommend to use [envchain](https://github.com/sorah/envchain). 
It enables to handle multiple AWS credentials easily and securely :key: 

Like this:

```bash
$ envchain --set personal AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY
$ envchain --set company AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY

# w/ personal account profile
$ envchain personal terraforming s3

# w/ company account profile
$ envchain company terraforming s3
```

Otherwise if you're using another operating system, I don't have any good idea...

**Comment by jen20** [comment]
The official AWS CLI already has a method for handling this via profiles - as I understand it from [the Ruby SDK doc here](http://docs.aws.amazon.com/sdkforruby/api/index.html) the Ruby client will attempt by default to use the the shared credentials file as a second preference to environment variables.

To support profiles it looks at first glance necessary to use an instance of [these SharedCredentials](http://docs.aws.amazon.com/sdkforruby/api/Aws/SharedCredentials.html).

Happy to look at pull requesting this functionality if it's likely to be accepted.

**Comment by dtan4** [comment]
`--profile` option was implemented at #135 and released as v0.4.0 :shipit: 
Please try this :exclamation:
